doozer/doozerlib/cli/inspect_stream.py

```diff
@@ -17,10 +17,7 @@
 @click.pass_obj
 async def inspect_stream(runtime, code, strict):
     code = AssemblyIssueCode[code]
-    if runtime.assembly != 'stream':
```

Comment:
This command is very useful in debugging non-stream assembly issues involving rhcos (a very small validation subset of release:gen-payload)